DependencyInjection/SkrzBunnyExtension.php

```diff
@@ -29,7 +29,7 @@ public function getConfigTreeBuilder()
 		$rootNode->children()->scalarNode("heartbeat")->defaultValue(60);
 
 		/** @var ArrayNodeDefinition $exchangesNode */
-		$exchangesNode = $rootNode->children()->arrayNode("exchanges")->normalizeKeys(false)->defaultValue([])->prototype("array");
+		$exchangesNode = $rootNode->children()->arrayNode("exchanges")->useAttributeAsKey("exchanges")->normalizeKeys(false)->defaultValue([])->prototype("array");
```

Comment:
I think this won't work. Haven't tested it yet, but here should be used "name" instead of exchanges.
At least based on this symfony documentation: https://symfony.com/doc/current/components/config/definition.html#array-node-options there should be used name or any children attribute.